core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -385,6 +387,8 @@ object KafkaConfig {
   val MaxConnectionsPerIpDoc = "The maximum number of connections we allow from each ip address"
   val MaxConnectionsPerIpOverridesDoc = "Per-ip or hostname overrides to the default maximum number of connections"
   val ConnectionsMaxIdleMsDoc = "Idle connections timeout: the server socket processor threads close the connections that idle more than this"
+  /************* Rack Configuration **************/
+  val RackDoc = "Rack of the broker"
```

Comment:
I think it would be good if we could elaborate a little more on the purpose of this property.